src/rpc.rs

```diff
@@ -106,7 +106,7 @@ impl RpcSol for RpcSolImpl {
         let signature = Signature::new(&signature_vec);
         meta.request_processor.get_signature_status(signature)
     }
-    fn get_balance(&self, meta: Self::Metadata, id: String) -> Result<(String, i64)> {
+    fn get_balance(&self, meta: Self::Metadata, id: String) -> Result<i64> {
         let pubkey_vec = match bs58::decode(id).into_vec() {
```

Comment:
While you're in the neighborhood, `map_err(|_| Error::invalid_request())`

And delete commented out code below?

Reply:
> And delete commented out code below?

You mean `send_transaction`?

Reply:
Oh yeah, I didn't click expand enough times in the github UI. All good.

Reply:
`map_err` applied, much nicer